app/routes.py

```diff
@@ -1,2 +1,247 @@
-from flask import Blueprint
+import re
+from flask import request, Blueprint
+from app import db
+from flask import jsonify
+from .models.customer import Customer
+from .models.video import Video
+from .models.rental import Rental
 
+import requests
+import os
+
+customer_bp = Blueprint("customers", __name__, url_prefix="/customers")
+video_bp = Blueprint("videos", __name__, url_prefix="/videos")
+rental_bp = Blueprint("rental", __name__, url_prefix="/rentals")
+
+
+@customer_bp.route("", methods=["POST"], strict_slashes=False)
+def add_customer():
+    request_body = request.get_json()
+    if "name" not in request_body or "postal_code" not in request_body or "phone" not in request_body:
+        return jsonify({
+        "details": "Invalid data"
+        }), 400
+    new_customer = Customer(name=request_body["name"],
+                            postal_code=request_body["postal_code"],
+                            phone=request_body["phone"])
+    
+    db.session.add(new_customer)
+    db.session.commit()
+    return new_customer.to_json(), 201
+
+@customer_bp.route("", methods=["GET"], strict_slashes=False)
+def customer_index():
+    customers = Customer.query.all()
+    customer_response = []
+    for customer in customers:
+        customer_response.append(customer.to_json())
+
+    return jsonify(customer_response), 200
+
+@customer_bp.route("/<customer_id>", methods=["GET"], strict_slashes=False)
+def get_customer(customer_id):
+    customer = Customer.query.get(customer_id)
+    if customer:
+        return jsonify(customer.to_json()), 200
+    return "", 404
```

Comment:
You can use `get_or_404` to simplify this:

```suggestion
    customer = Customer.query.get_or_404(customer_id)

    return jsonify(customer.to_json()), 200
```